extensions/clusters/kubernetesversions/all.go

```diff
@@ -254,10 +228,6 @@ func sortReleases(rancherVersion *semver.Version, releases []interface{}) (allAv
 			continue
 		}
 
-		if !rancherVersion.GreaterThan(minRancherVersion) && !rancherVersion.LessThan(maxRancherVersion) {
-			continue
-		}
```

Comment:
Priya just a small thing about this, we still want to use `minRancherVersion` so we need to ensure the current rancher version is greater than the `minRancherVersion`, so something like this:

		if !rancherVersion.GreaterThan(minRancherVersion) {
			continue
		}

Reply:
@caliskanugur I think this may not be needed as well. Based on what I have noticed, what gets listed with the endpoint are the compatible versions for that specific rancher version. If the min version is less than the rancher version , that wouldn't be listed

Reply:
You're right, this is also related to @anupama2501 's min version question, so from Priya's previous confirmation with the KDM team turns out we don't need `minRancherVersion` at all either. Resolving this one.